lib/parity/backup.rb

```diff
@@ -62,8 +62,13 @@ def remote_db_backup_url
     end
 
     def development_db
+      YAML.load(parsed_database_yml)['development']['database']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.